### ISSUE TITLE
Liquid in new chunks does not spread.

### ISSUE THREAD

**Issue by TheLemzz** [HimmelKreis4865/JavaGen#3]
![image](https://github.com/HimmelKreis4865/JavaGen/assets/75723466/8370e5bc-00bd-4342-8cde-06f3c38c2fd6)
![image](https://github.com/HimmelKreis4865/JavaGen/assets/75723466/c7b2165c-5351-4fd9-9920-d17d4295fd78)

Thats critical issue

**Comment by TheLemzz** [comment]
Some liquids could spread, but some did not

**Comment by HimmelKreis4865** [comment]
yeah this depends on the java generation. To reduce lags and time efforts to generate the chunk I didn't yet choose to make them flow again, but I will look at this in the near future

**Comment by TheLemzz** [comment]
It will be very cool if you add this. These frozen liquids look extremely unnatural
